docs/advanced_topics/customisation/extending_draftail.rst

```diff
@@ -310,7 +310,7 @@ Finally, we register the JS components of our plugin:
         decorator: Stock,
     });
 
-And that’s it! All of this setup will finally produce the following HTML on the site’s front-end:
+And that’s it! All of this set up will finally produce the following HTML on the site’s front-end:
```

Comment:
I'd consider the word "setup" to be a similar thing to "login": one word when used as a noun ("this will be installed during setup"), but two words when used as a verb ("you need to set up Elasticsearch").